cmd/agent/app/integrations.go

```diff
@@ -0,0 +1,269 @@
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build cpython
+
+package app
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/executable"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+const (
+	constraintsFile = "agent_requirements.txt"
+	tufConfigFile   = "public-tuf-config.json"
+	tufPyPiServer   = "https://integrations-core-wheels.s3.amazonaws.com/simple/"
+	pyPiServer      = "https://pypi.org/simple/"
+)
+
+var (
+	allowRoot    bool
+	withoutTuf   bool
+	useSysPython bool
+	tufConfig    string
+)
+
+func init() {
+	AgentCmd.AddCommand(tufCmd)
```

Comment:
A `List`/`freeze` command would be very useful (if possible) !

Reply:
Done!